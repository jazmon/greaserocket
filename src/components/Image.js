// @flow
import React from 'react';
import { View, Text, StyleSheet, Image as RNImage } from 'react-native';

const defaultProps = {};

type Props = {};

const Image = ({ ...props }: Props): React.Element<*> => <RNImage {...props} />;

Image.defaultProps = defaultProps;

const styles = StyleSheet.create({});

export default Image;
