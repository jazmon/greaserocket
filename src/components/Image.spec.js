import 'react-native';
import React from 'react';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

import Image from './Image';

it('renders correctly', () => {
  const tree = renderer.create(<Image />);
  expect(tree).toMatchSnapshot();
});
