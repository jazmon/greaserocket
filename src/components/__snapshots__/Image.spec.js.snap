// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={null}
>
  <Image
    onLoadEnd={[Function]}
    onLoadStart={[Function]}
  />
</View>
`;
